repose-aggregator/components/filters/keystone-v2/src/main/scala/org/openrepose/filters/keystonev2/KeystoneV2Filter.scala

```diff
@@ -626,6 +626,7 @@ class KeystoneV2Filter @Inject()(configurationService: ConfigurationService,
       // Removes an extra slash at the end of the URI if applicable
       val serviceUri = keystoneV2Config.getIdentityService.getUri
       keystoneV2Config.getIdentityService.setUri(serviceUri.stripSuffix("/"))
+      akkaServiceClient.setConnectionPoolId(keystoneV2Config.getIdentityService.getConnectionPoolId)
```

Comment:
i think this client might be one of the per node services, so there could be a couple with the right config. or the update service could be randomly forcing multiple reloads of the main context